### ISSUE TITLE
Remove matplotlib dependency

### ISSUE THREAD

**Issue by jmespadero** [jmespadero/cityGen#11]
cityGen2D.py use matplotlib module to generate .svg images which show debug information of each step of 2D map. The problem is that python interpreter bundled with blender does not include matplotlib. 

We can study to remove that dependency by creating .svg files using pure python, so they can be generated too when executed inside of blender.

**Comment by jmespadero** [comment]
This is a sample of a pretty printed SVG file with two polygons:
```
<svg width="300" height="250">
  <polygon points="150,0 75,100 225,100" style="fill:#ffeeaa;stroke:black;stroke-width:2" />
  <polygon points="150,200 75,100 225,100" style="fill:#ffeeaa;stroke:black;stroke-width:2" />
  <text x="150" y="50" fill="red" text-anchor="middle">Hello!</text>  
</svg>
```
This is the same, but using [path](https://www.w3schools.com/graphics/svg_path.asp) instead of [polygon](https://www.w3schools.com/graphics/svg_polygon.asp)
```
<svg width="300" height="250">
  <path d="M150 0 L75 100 L225 100 Z" fill="#ffeeaa" stroke="#000" stroke-width="2" stroke-linejoin="round" />
  <path d="M150 200 L75 100 L225 100 Z" fill="#ffeeaa" stroke="#000" stroke-width="2" stroke-linejoin="round" />  
</svg>
```

Basically this is all you need to star drawing 2D diagrams

**Comment by jmespadero** [comment]
This issue is solved with commit 42c23ecb
CityGen2D.py now uses an internal SVG render by default. 
You can still force render with matplotlib using the new argument --matplotlib 

![city map verbose](https://user-images.githubusercontent.com/6451497/30691341-b97e25b2-9ec7-11e7-9e92-7b2a4d890dec.png)
